TPP.Core/Commands/Definitions/HelpCommand.cs

```diff
@@ -220,7 +220,6 @@ private CommandResult Execute(CommandContext context)
             ["pinball"] = "Play pinball, or use a subcommand. Details: https://pastebin.com/W4Vrwvmx",
             ["pinballodds"] = "Display the payout rate of pinball in the past timeframe (default 90 days), with a specified score on a specified table. Arguments: <time_to_go_back>(optional), <score>(optional), <table>",
             ["tutorial"] = "Start the tutorial, if it has not already triggered.",
-            ["bet"] = "Chat only: bet on PBR using Pokéyen. Arguments: <team> <amount or percent>",
```

Comment:
how is this going to work with dual core compatibility? if you're removing the help message it will not return anything in dual core mode, where we have not added the command afaik

Reply:
very good point, I'll keep it